### PR TITLE
Update apex-tmlanguage to 1.0.2

### PR DESCRIPTION
**Pull request by vazexqi**
### What does this PR do?

Update apex-tmlanguage to 1.0.2

### What issues does this PR fix or reference?

@W-4997695, W-4998665, W-4998669@

### CODE REVIEW

**Comment by vazexqi** [comment]
I QA'ed @lcampos changes to the grammar and found that the changes to be low-risk and high-value. Therefore, I would like to just fold them into the release/v43.3.0 branch.

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 ### Fixed
 #### salesforcedx-vscode-apex
 
-* Improve syntax highlighting for built-in Apex classes, methods, and types ([PR #474](https://github.com/forcedotcom/salesforcedx-vscode/pull/474))
+* Improve syntax highlighting for built-in Apex classes, methods, and types ([PR #474](https://github.com/forcedotcom/salesforcedx-vscode/pull/474), [PR #484](https://github.com/forcedotcom/salesforcedx-vscode/pull/484))
```
Comment (lcampos) [inline packages/salesforcedx-vscode/CHANGELOG.md]: Should we add another bullet point to highlight the improvements on soql syntax highlighting and for loops ?

Comment (vazexqi) [reply packages/salesforcedx-vscode/CHANGELOG.md]: I think it's fine. I was lumping it as highlighting for Apex clases, methods, and types.

Comment (lcampos) [reply packages/salesforcedx-vscode/CHANGELOG.md]: :shipit:

**Comment by vazexqi** [comment]
This can go in now since this is just a quick change. But, the build will continue failing until #485 goes in.

**Pull request merged**